guides/partials.md

```diff
@@ -4,4 +4,33 @@ layout: guides
 permalink: /guides/partials/
 ---
 
-#Partials
+## Partials
+
+A Dust partial promotes "DRY" behavior for reusable/shared markup. Partials themselves can be composed of one or more partials. Partials rely on the JSON/context of the parent template invoking it. Lets peek under the covers to see how a Dust partial works.
+
+You can have multiple .dust files and reference one Dust template as part of another one. This is the basis for components or reusable templates for tasks like a common header and footer on multiple pages. Note that the .dust file extension is used here in examples but .tl is also commonly seen. Since it only matters to the build process you can use whatever extension works for you.
```

Comment:
You should surround file extensions (and anything else that is code like) with backtic's. e.g. \`.dust\`.